### ISSUE TITLE
Need help on checking my summary of ZeroNet

### ISSUE THREAD

**Issue by deavmi** [HelloZeroNet/ZeroNet#592]
I am writing a page on ZeroNet for LibreNet SA's wiki and would like to know if the information I have written on ZeroNet is correct in terms of the system.

https://wiki.librenet.co.za/doku.php?id=librenetsa:networking:decentralized:zeronet

You can easily sign up for an account on the wiki and make your changes as you see fit.

Also I need the description to be relatively in laymans terms.

**Comment by deavmi** [comment]
Also I need the description to be relatively in laymans terms.

**Comment by rllola** [comment]
I have this message : 
```
You've followed a link to a topic that doesn't exist yet. If permissions allow, you may create it by clicking on “Create this page”.
```

**Comment by deavmi** [comment]
Sorry. Try the new link.

**Comment by MuxZeroNet** [comment]
@deavmi Why not make a GitHub repo so that we can send pull requests?

**Comment by imachug** [comment]
@HelloZeroNet This is outdated and isn't actually related to ZeroNet development, and we have other issues about docs too. Can you close this please?
